list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'enumerable'
+# class with a enumerable methods
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each
+    yield(@list)
+  end
```

Comment:
- [ ] Let's try and test this `each` method. If you wrote the following code:
```ruby
list = MyList.new(1, 2, 3, 4)

list.each do |e| 
  p(e)
end
```
- This should return each element in the array like so:
```ruby
1
2
3
4
```

- However, you'll find that it returns this code:
```ruby
[1, 2, 3, 4]
```

- This happens because the `each` method you implemented, only yields one item which is the whole array. In this case, `yield` is treated as the normal `return` because it's yielding just one time. Kindly fix this method to yield each item in the array instead of yielding it all at once. You can test the first code snippet I provided and see if the outcome matches the correct answer.